site/content/en/docs/Getting started/first_kustomization.md

```diff
@@ -57,10 +107,259 @@ spec:
         app: nginx
     spec:
       containers:
-      - name: nginx
-        image: nginx:latest
+      - image: nginx
+        name: nginx
         ports:
         - containerPort: 80
+```
+
+### What are Bases and Overlays?
+
+Let's think of a scenario where we need to deploy the nginx manifests from the previous section to two environments called `Staging` and `Production`. The manifests for these two environments will be mostly identical, with only a few minor changes between them. Traditionally to achieve these changes, we could duplicate the manifests and apply the changes manually or rely on some templating engine. With kustomize we can avoid templating and duplication of our manifests and apply the different changes we need using overlays. With this approach, the `base` would contain the common part of the our manifests and the `overlays` contain our environment specific changes.
+
+To continue with this tutorial, create the `kustomization.yaml` files for our two overlays:
+
+```bash
+cat <<'EOF' >overlays/staging/kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../../base
 EOF
+
+cat <<'EOF' >overlays/production/kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../../base
```

Comment:
Do you think it's worth adding a sentence somewhere about how bases can be specified in the same field as other resources?

Reply:
Yes, will add that

Reply:
Since this is a tutorial, I'd actually avoid explaining things we aren't demonstrating through the walkthrough itself. That said, we could have a section on adding a resource to one of the variants.